File: 백준/Silver/10431. 줄세우기/줄세우기.py
import sys
input = sys.stdin.readline
t = int(input())
tall = []
for _ in range(t):
    tall= list(map(int,input().split()))
    total = 0
    for i in range(1,len(tall)-1):
        for j in range(i+1, len(tall)):
            if tall[i]>tall[j]:
                tall[i],tall[j] = tall[j],tall[i]
                total+=1
    print(tall[0], total)

File: 백준/Silver/10431. 줄세우기/줄세우기.java
import java.util.*;
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int t = Integer.parseInt(br.readLine());
        StringBuilder sb = new StringBuilder();
        for(int k=0;k<t;k++){
            String[] input  = br.readLine().split(" ");
            int stn = Integer.parseInt(input[0]);
            int[] heights = new int[input.length-1];
            for(int i=1;i<input.length;i++){
                heights[i-1] = Integer.parseInt(input[i]);
            }
            int total = 0;
            for(int i=0;i<heights.length-1;i++){
                for(int j=i+1;j<heights.length;j++){
                    if(heights[i]>heights[j]){
                        int tmp = heights[i];
                        heights[i] = heights[j];
                        heights[j] = tmp;
                        total++;
                    }
                }
            }
            sb.append(stn).append(" ").append(total).append("\n");
        }
        System.out.print(sb);
    }
}